### PR TITLE
CHANGELOG.md changes for 2.0.5 release

### PR DESCRIPTION
**Pull request by rohanbhattacharjee**
Please review if this change log captures all the changes we did for this release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,18 @@ All notable changes to this project will be documented in this file.
...
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374)
+
+### Fixed
+- Fixed bug wherein policy was not destroyed and recreated when compartment is changed (#389)
+- Fixed errors with terraform import because of missing vcn_id in `*.tfstate` files (internet_gateway, route_tables, dhcp_options) (#388, #379)
+
+### Notes
```
Comment (briangustafson) [inline CHANGELOG.md]: Also copy over the 2 notes that we've been putting in every release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,10 +3,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unreleased
+## 2.0.5 - 2017-12-14
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374)
```
Comment (briangustafson) [inline CHANGELOG.md]: Would be useful to link to the new doc for this, https://github.com/oracle/terraform-provider-oci/blob/master/docs/Managing%20Default%20Resources.md

File: CHANGELOG.md (hunk 1 above)
Comment (rohanbhattacharjee) [reply CHANGELOG.md]: done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,10 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unreleased
+## 2.0.5 - 2017-12-14
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374). See more details about this feature [here]()https://github.com/oracle/terraform-provider-oci/blob/master/docs/Managing%20Default%20Resources.md)
```
Comment (briangustafson) [inline CHANGELOG.md]: That doesn't show up quite right - click on "view".

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,10 +3,20 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unreleased
+## 2.0.5 - 2017-12-14
 
 ### Added
 - Support for security list rule source port ranges (#340). This can be specified in "tcp_options" and "udp_options" using "source_port_range".
+- Allow configuration of default resources under VCNs (#374). See more details about this feature [here](https://github.com/oracle/terraform-provider-oci/blob/master/docs/Managing%20Default%20Resources.md)
+
+### Fixed
```
Comment (rcohenma) [inline CHANGELOG.md]: Add fix for issue 170 https://github.com/oracle/terraform-provider-oci/commit/ea9e254bfcb50f38ca8faccef52e8daa583dffbb#diff-986abfb66fbe4ec940da975ae72b749b

"Fix error where same retry token was being used for multiple requests in some development environments when auto retries were activated (#170)"

**Pull request merged**